scripts/translation/xliffmerger.py

```diff
@@ -40,8 +47,9 @@ def update_trans_units(body_element, trans_units, id_new_value_dict):
 
 
 def process(original_file, translation_file):
-    xml_tree_original = ET.parse(original_file)
-    xml_tree_translation = ET.parse(translation_file)
+    parser = ET.XMLParser(target=CommentedTreeBuilder())
```

Comment:
Using the same instance of `parser` two parse both files does not seem to work for me

```
INFO:root:
processing StringResource_ca.xml
Traceback (most recent call last):
  File "/home/pascal/git/uyuni/scripts/translation/xliffmerger.py", line 123, in <module>
    process(original, translation)
  File "/home/pascal/git/uyuni/scripts/translation/xliffmerger.py", line 53, in process
    xml_tree_original = ET.parse(original_file, parser=parser)
  File "/usr/lib64/python3.6/xml/etree/ElementTree.py", line 1196, in parse
    tree.parse(source, parser)
  File "/usr/lib64/python3.6/xml/etree/ElementTree.py", line 603, in parse
    parser.feed(data)
xml.etree.ElementTree.ParseError: parsing finished: line 23921, column 8
```

Reply:
Creating two parsers seems to work
```
parser1 = ET.XMLParser(target=CommentedTreeBuilder())
parser2 = ET.XMLParser(target=CommentedTreeBuilder())
xml_tree_original = ET.parse(original_file, parser=parser1)
xml_tree_translation = ET.parse(translation_file, parser=parser2)
```
I have no idea why though